source/io/module_httpClient.js

```diff
@@ -573,7 +573,8 @@
                 method: method,
                 url: url,
                 xhrTimeout: xhrTimeout,
-                buffer: buffer
+                buffer: buffer,
+                withCredentials: httpClient.withCredentials
```

Comment:
withCredentials is a property of an HttpClient instance, it should not be passed in here